tests/integration/api/test_extra_var.py

```diff
@@ -41,6 +51,21 @@ def test_create_extra_var(client: APIClient):
     assert models.ExtraVar.objects.filter(pk=id_).exists()
 
 
+@pytest.mark.django_db
+def test_create_vault_extra_var(client: APIClient):
+    data_in = {
+        "extra_var": TEST_VAULT_EXTRA_VAR,
+    }
+    response = client.post(f"{api_url_v1}/extra-vars/", data=data_in)
+    assert response.status_code == status.HTTP_201_CREATED
+    id_ = response.data["id"]
```

Comment:
@hsong-rh Is there a reason for a trailing underscore?

Reply:
@hsong-rh Can we fetch the limit from the REST API call and ensure it made a round trip.

Reply:
changed